.gitignore

```diff
@@ -1,11 +0,0 @@
-/target
```

Comment:
Why was this file removed? (`git checkout master .gitignore` should restore it if this was removed by accident.)

Reply:
This comment still applies.